e2e/fixtures/fdb_operator_fixtures.go

```diff
@@ -288,3 +288,8 @@ func WithLocalitiesForExclusion(_ *Factory, cluster *fdbv1beta2.FoundationDBClus
 func WithUnifiedImage(_ *Factory, cluster *fdbv1beta2.FoundationDBCluster) {
 	cluster.Spec.UseUnifiedImage = pointer.Bool(true)
 }
+
+// WithPodIPFamily is an option that sets the Pod IP Family for a cluster.
+func WithPodIPFamily(_ *Factory, cluster *fdbv1beta2.FoundationDBCluster) {
```

Comment:
The name of the method is a bit misleading, as we are setting the Pod IP family to IPv6 always, we could also set this to IPv4 if we wanted. So probably `WithPodIPFamilyIPv6` is a better name.